_data/shared_settings.yml

```diff
@@ -0,0 +1,18 @@
+# These are the values sent in the form to the contact center
+# They should *not* be translated because they are keys for the backend
+contact_page:
+  agencies:
+    cbp_jobs: Customs and Border Protection Jobs
+    cbp_roam: Customs and Border Protection ROAM
+    dot_fmcsa: DOT FMCSA (Motor Carrier Safety)
+    dot_nrcme: DOT National Registry of Certified Medical Examiners
+    ds_logon: DS Logon
+    move_mil: DOD Moves.mil (Homes or Office)
```

Comment:
one note, this is *different* than the English copy we display:
`DOD Move.mil (Homes or Office)` (plural vs singular)